classes/class.tapestry-node.php

```diff
@@ -424,6 +424,21 @@ public function addReview($comments)
         ];
     }
 
+    public function getMediaType($node)
```

Comment:
Actually, these properties are private and thus cannot be directly accessed. They did this so that they can access the properties through a public function. It is a bit confusing, but it may be necessary in this case.

Reply:
That is rather unfortunate but I see why it's needed. Thanks for looking into this!